### PR TITLE
Bug 2057972: Switch to "verb kind" phrase so that Chinese translations do not have spaces between.

### PR DESCRIPTION
**Pull request by sg00dwin**
Fix bug https://bugzilla.redhat.com/show_bug.cgi?id=2057972

### CODE REVIEW

File: frontend/public/components/RBAC/bindings.jsx [hunk 1]
```diff
@@ -81,9 +81,7 @@ const menuActions = ({ subjectIndex, subjects }, startImpersonate) => {
 
   const actions = [
     (kind, obj) => ({
-      label: i18next.t('public~Duplicate {{kindLabel}}', {
-        kindLabel: getKindLabel(kind),
-      }),
+      label: i18next.t('public~Duplicate RoleBinding'),
```
Comment (rhamilto) [inline frontend/public/components/RBAC/bindings.jsx:84]: This will result in the label for non-RoleBindings being labeled "RoleBinding", no?

Comment (sg00dwin) [reply frontend/public/components/RBAC/bindings.jsx:84]: Would this action have a label other than "RoleBinding"?

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Pull request merged**